### PR TITLE
Release v0.10.1

### PR DESCRIPTION
**Pull request by jhkennedy**
I went through the golden tests and confirmed everything we were requested and expected to change in the metadata has been changed. Should be g2g!

### CODE REVIEW

File: hyp3_autorift/vend/README.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ We've replaced it  with `hyp3_autorift.io.get_topsinsar_config`.
    * Set the fallback value of `stable_shift` netCDF attribute to `0` instead `np.nan`
    These changes have been [proposed upstream](https://github.com/nasa-jpl/autoRIFT/pull/73) and should be applied
    in the next `nasa-jpl/autoRIFT` release.
-1. The changes listed in `CHANGES-183.diff` were applied in [ASFHyP3/hyp3-autorift#183](https://github.com/ASFHyP3/hyp3-autorift/pull/183)
+2. The changes listed in `CHANGES-183.diff` were applied in [ASFHyP3/hyp3-autorift#183](https://github.com/ASFHyP3/hyp3-autorift/pull/183)
    to correctly set the netcdf `img_pair_info:correction_level_img` attribute values for Sentinel-2 scenes after the transition from
    Earth Search COG IDs to ESA IDs.
+3. The changes listed in `CHANGES-189.diff` and `CHANGES-191.diff` were applied in 
+   [ASFHyP3/hyp3-autorift#189](https://github.com/ASFHyP3/hyp3-autorift/pull/189)
+   and [ASFHyP3/hyp3-autorift#180](https://github.com/ASFHyP3/hyp3-autorift/pull/180)
```
Comment (asjohnston-asf) [inline hyp3_autorift/vend/README.md:43]:
```suggestion
   and [ASFHyP3/hyp3-autorift#191](https://github.com/ASFHyP3/hyp3-autorift/pull/191)  
```

**Comment by jhkennedy** [comment]
flake8 failure is being tracked here:
https://github.com/ASFHyP3/actions/issues/44

We can ignore it for this release

File: hyp3_autorift/vend/netcdf_output.py [hunk 2]
```diff
@@ -13,6 +13,21 @@
 import hyp3_autorift
 
 
+def get_satellite_attribte(info):
+    mission_mapping = {
+        'L': 'Landsat',
+        'S': 'Sentinel',
```
Comment (asjohnston-asf) [inline hyp3_autorift/vend/netcdf_output.py:19]:
```suggestion
        'L': 'Landsat ',
        'S': 'Sentinel-',
```

File: hyp3_autorift/vend/netcdf_output.py [hunk 3]
```diff
@@ -13,6 +13,21 @@
...
+    }
+
+    satellite_1 = f'{mission_mapping[info["mission_img1"]]} {info["satellite_img1"]}'
+    satellite_2 = f'{mission_mapping[info["mission_img2"]]} {info["satellite_img2"]}'
```
Comment (asjohnston-asf) [inline hyp3_autorift/vend/netcdf_output.py:23]:
```suggestion
    satellite_1 = f'{mission_mapping[info["mission_img1"]]}{info["satellite_img1"]}'
    satellite_2 = f'{mission_mapping[info["mission_img2"]]}{info["satellite_img2"]}'
```

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**